create-hpc-cluster-custom-image/azuredeploy.json

```diff
@@ -2,23 +2,17 @@
   "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
   "contentVersion": "1.0.0.0",
   "parameters": {
-    "namePrefix": {
+    "clusterName": {
```

Comment:
yes, it has been a long time since the template create-hpc-cluster-custom-image was refreshed last time because it is low priority by comparing with other two templates (create-hpc-cluster and create-hpc-cluster-linux-cn). We finally have time to refresh it now.